infra/service.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hospital-users-service
+spec:
+  type: NodePort
```

Comment:
why it's `NodePort`? can be LoadBalancer or ClusterIP

Reply:
agree, I think it's better to use ClusterIP here